src/command.c

```diff
@@ -564,7 +564,19 @@ static bool cmd_rtt(target_s *t, int argc, const char **argv)
 			if (rtt_ident[i] == '_')
 				rtt_ident[i] = ' ';
 		}
-	} else if (argc == 2 && strncmp(argv[1], "ram", command_len) == 0)
+	} else if (argc == 3 && strncmp(argv[1], "down", command_len) == 0) {
```

Comment:
Perhaps instead of "down" lets use "send" - "down" doesn't make much grammatical or linguistical sense to us as a native English speaker and leaves us wondering what this really does. `mon rtt send <stuff>` makes much more sense - you're sending <stuff> to the target.